apps/src/templates/manageStudents/DownloadParentLetter.jsx

```diff
@@ -4,16 +4,18 @@ import Button from '@cdo/apps/templates/Button';
 import i18n from '@cdo/locale';
 import firehoseClient from '@cdo/apps/lib/util/firehose';
 import {teacherDashboardUrl} from '@cdo/apps/templates/teacherDashboard/urlHelpers';
+import {ParentLetterEntryPoint} from '@cdo/apps/templates/manageStudents/manageStudentsRedux';
 
 const styles = {
   button: {
     marginLeft: 'auto'
   }
 };
 
-export default class DownloadParentLetters extends Component {
+export default class DownloadParentLetter extends Component {
   static propTypes = {
-    sectionId: PropTypes.number
+    sectionId: PropTypes.number,
+    buttonLocation: PropTypes.oneOf(Object.values(ParentLetterEntryPoint))
```

Comment:
nit - I'd consider changing 'buttonLocation' and 'ParentLetterEntryPoint' to names that reflect that these variables are used for metrics. As is, they make me think that they somehow affect how the button will be desplayed with regards to the location.